deploy/examples/walkthrough.yaml

```diff
@@ -2,3 +2,8 @@ apiVersion: "integreatly.aerogear.org/v1alpha1"
 kind: "Walkthrough"
 metadata:
   name: "example"
+spec:
+  namespace: example
+  username: developer
+  services:
+   - fuse
```

Comment:
**namespace** = The target namespace where the services will be provisioned (this should possibly be generated by the operator instead of being specified)
**username** = The username of the user who initiated the walkthough from the launcher app
**services** = A list of services required by the walkthrough (structure might changes since we probably want to save some state  for each service so we know if it's provisioned or not)